src/lib.rs

```diff
@@ -567,6 +581,61 @@ impl FrankaSim {
                 ..Default::default()
             },
         );
+        let mut robot_joint_indices: [Option<usize>; 7] = [None; 7];
```

Comment:
Use `PANDA_NUM_DOFS` here instead of magic number

Reply:
done